src/cinnamon-app-system.c

```diff
@@ -897,6 +1013,26 @@ cinnamon_app_system_lookup_startup_wmclass (CinnamonAppSystem *system,
   return g_hash_table_lookup (system->priv->startup_wm_class_to_app, wmclass);
 }
 
+/**
+ * cinnamon_app_system_lookup_startup_wmclass_for_flatpak_app:
+ * @system: a #CinnamonAppSystem
+ * @wmclass: (nullable): A WM_CLASS value
```

Comment:
this probably shouldn't be labeled nullable (or optional) if we just return NULL when it is NULL